pkg/postgres/studentStore.go

```diff
@@ -47,7 +49,27 @@ func (s StudentStore) GetObservations(studentId string) ([]Observation, error) {
 	}
 	return observations, nil
 }
+func (s StudentStore) CheckPermissions(studentId string, userId string) (bool, error) {
+	var student Student
 
+	if err := s.Model(&student).
+		Relation("School").
+		Relation("School.Users", func(q *orm.Query) (*orm.Query, error) {
+			return q.Where("user_id = ?", userId), nil
+		}).
+		Where("student.id=?", studentId).
+		Select(); err == pg.ErrNoRows {
+		return false, nil
+	} else if err != nil {
+		return false, err
```

Comment:
We should wrap the error here too. So that we'll get its stack trace when logged. Similar to this maybe:
```suggestion
		return false, richErrors.Wrap(err, "failed checking user access to student")
```